brains/list/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .alch import alch
+from .misc import misc
+from .other import other
+from .stock import stock
```

Comment:
github informs me this is missing a newline at the end of the file

Reply:
fixed in 7177f07